packages/website/pages/pricing_old.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react';
+
+import TiersPageData from '../content/pages/pricing.json';
+import Scroll2Top from '../components/scroll2top/scroll2top.js';
+import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
+import { initFloaterAnimations } from '../lib/floater-animations.js';
+
+export default function Home() {
+  const sections = TiersPageData.page_content;
+  const animations = TiersPageData.floater_animations;
+
+  useEffect(() => {
+    let pageFloaters = {};
+    if (animations) {
+      initFloaterAnimations(animations).then(result => {
+        pageFloaters = result;
+      });
+    }
+    return () => {
+      if (pageFloaters.hasOwnProperty('destroy')) {
+        pageFloaters.destroy();
+      }
+    };
+  }, [animations]);
+
+  return (
+    <>
+      <main className="page page-pricing">
+        {sections.map((section, index) => (
+          <BlockBuilder id={`pricing_section_${index + 1}`} key={`section_${index}`} subsections={section} />
+        ))}
+      </main>
+
+      <Scroll2Top />
+    </>
+  );
+}
+
+export function getStaticProps() {
```

Comment:
Is this used?

Reply:
Ah, this should've been a checkbox: delete old pricing stuff